app/vmalert/README.md

```diff
@@ -119,6 +119,12 @@ name: <string>
 # `eval_offset` can't be bigger than `interval`.
 [ eval_offset: <duration> ]
 
+# Optional
+# Adjust the `time` parameter of group evaluation requests to match intentional query delay from datasource.
+# This will override global `-rule.evalDelay` flag if both be using.
+# See https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5155.
+[ eval_delay: <duration> ]
```

Comment:
Should we have a default value here? For example, 30s default delay will compensate default latency offset

Reply:
Good idea, set to 30s now.